spec/lib/game_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'game'
+
+describe Game do
+  let(:player) { double :player, name: 'foo', choice: 'Scissors' }
+  before(:each) { @game = Game.new(player) }
```

Comment:
What is the difference in practice between before(:each) and let?

Reply:
That's a great question and I'm not sure - I think they probably behave the same? Although my habits are to only declare doubles in a `let`, and everything else that I want to use goes in a `before(:each)`. What do you think?